examples/timer-log/README.adoc

```diff
@@ -0,0 +1,27 @@
+= timer-log
+
+This is a basic hello world example that uses a Camel timer that
+triggers every second and prints to the log.
+
+To run it:
+
+[source,text]
+----
+$ mvn clean compile quarkus:dev -DnoDeps
+----
+
+Then look at the log output in the console. As we run the example
+in Quarkus Dev Mode, you can edit the source code and have live updates.
+For example try to change the logging output to be `Bye World`.
+
+== Native build
+
+To build as native:
+
+[source,text]
+----
+$ mvn package -Pnative-image
```

Comment:
`-Pnative-image` is correct, but in the user guide we say `-Dnative` which is equivalent and a bit shorter.

I would perhaps also be worthwhile to note that the above requires Docker.

The result is actually a Linux executable that won't run on Windows and Mac. Not sure we should go in such details here?

Reply:
Updated and the other example is using native-image so we can update those instead